src/Avalonia.Controls/ToolTipService.cs

```diff
@@ -36,7 +36,7 @@ private void InputManager_OnProcess(RawInputEventArgs e)
         {
             if (e is RawPointerEventArgs pointerEvent)
             {
-                if (e.Root == _tipControl?.GetValue(ToolTip.ToolTipProperty)?.PopupHost)
+                if (_tipControl?.GetValue(ToolTip.ToolTipProperty) is { } currentTip && (currentTip.IsPointerOver || e.Root == currentTip.PopupHost))
```

Comment:
Probably won't affect tooltips, but there is a hidden "special behavior" of IsPointerOver (as well as PointerEntered/Exited in general) - while element is captured, it always will be "pointer over".  See https://github.com/AvaloniaUI/Avalonia/pull/15357